typescript/common-resolvers/src/export/export-to-csv.fixtures.ts

```diff
@@ -0,0 +1,147 @@
+import { LabelType } from "@labelflow/graphql-types";
+import { PartialDeep } from "type-fest";
+import { Repository } from "..";
+import { DbDataset, DbImage, DbLabel, DbLabelClass } from "../types";
+
+const DATE = new Date("1995-12-17T03:24:00").toISOString();
+
+export const DATASET_DATA: DbDataset = {
+  id: "8f47e891-3b24-427a-8db0-dab362fbe269",
+  name: "Test Dataset",
+  slug: "test-dataset",
+  createdAt: DATE,
+  updatedAt: DATE,
+  workspaceSlug: "full-user-workspace",
+};
+
+const IMAGES_DATA: DbImage[] = [
+  {
+    id: "9b8fb142-8388-11ec-a8ee-f7dcb0508f86",
+    createdAt: DATE,
+    updatedAt: DATE,
+    name: "Test Image with labels 1",
+    url: "https://localhost:3000/api/downloads/8f47e891-3b24-427a-8db0-dab362fbe269/9b8fb142-8388-11ec-a8ee-f7dcb0508f86.jpg/original.jpeg",
+    path: "https://localhost:3000/api/downloads/8f47e891-3b24-427a-8db0-dab362fbe269/9b8fb142-8388-11ec-a8ee-f7dcb0508f86.jpg/original.jpeg",
+    thumbnail200Url:
+      "https://localhost:3000/api/downloads/8f47e891-3b24-427a-8db0-dab362fbe269/9b8fb142-8388-11ec-a8ee-f7dcb0508f86.jpg/t200.jpeg",
+    mimetype: "image/jpeg",
+    width: 1920,
+    height: 1080,
+  },
+  {
+    id: "a552037e-8388-11ec-965b-07327355c580",
```

Comment:
Can you merge both declarations in a map? 
![image](https://user-images.githubusercontent.com/2091902/155768794-a4c4b5c9-c2ee-4e09-92f2-6302b70aa0f9.png)